packages/components/src/components/tag/tag-styles.ts

```diff
@@ -8,7 +8,13 @@ import {
   pxToRemWithUnit,
   ThemedColors,
 } from '../../styles';
-import { fontStyle, fontWeight, textXSmallStyle } from '@porsche-design-system/utilities-v2';
+import {
```

Comment:
Isn't Anastasia working on those files?

Reply:
Yes, we aligned the changes. Focus styles should be the same for `tag` and `tag-dismissible`